samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js

```diff
@@ -315,11 +315,10 @@
                   elapsedTime = data[1];
                   lastAid = data[2];
                   if (totalTime === elapsedTime) {
-                      $("[id$=\\:save]")[0].click();
                       clearInterval(localCount);
                       clearInterval(ajaxCount);
                       $("#timeoutWarning").hide();
-                      leaveAssessment();
+                      $("[id$=\\:submitNoCheck]")[0].click();
```

Comment:
I'm not too familiar with this code. Conceptually it makes sense. Would it be appropriate to move this `submitNoCheck` click into the `leaveAssessment()` routine?

Reply:
i think leaveAssessment() is for when the assessment was already submitted by the back-end java timer